webrtc.html

```diff
@@ -2449,16 +2453,69 @@ <h4>
                               <li>
                                 <p>
                                   If a suitable transceiver was found
-                                  (<var>transceiver</var> is set),
+                                  (<var>transceiver</var> is set), and
                                   <var>proposedSendEncodings</var> is non-empty,
-                                  and
-                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
-                                  is empty, then set
-                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
-                                  to <var>proposedSendEncodings</var>, and set
-                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastReturnedParameters]]}}
-                                  to <code>null</code>.
+                                  run the following steps:
                                 </p>
+                                <ol>
+                                  <li class="needs-test">
+                                    <p>
+                                      If the length of
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                                      is `1`, and the lone encoding
+                                      [=map/contains=] no
+                                      {{RTCRtpCodingParameters/rid}} member, set
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                                      to <var>proposedSendEncodings</var>.
+                                    </p>
+                                  </li>
+                                  <li class="needs-test">
+                                    <p>
+                                      Assert: The first encoding of
```

Comment:
We haven't used "Assert:" language before; I'd suggest saying "At this point, " instead.

Reply:
I wasn't aware this convention existed, but I confirm it's one that is specifically [supported by ReSpec](https://github.com/w3c/respec/issues/1762), supported in other specs and defined in [Infra](https://infra.spec.whatwg.org/#assertions)